src/navigation/TabNavigation.tsx

```diff
@@ -0,0 +1,39 @@
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import {HomeScreen} from 'screens';
+import SettingsNavigation from './SettingsNavigation';
+import { blueColors, grayColors } from 'styles/themeColors';
+
+const TabNavigation = () => {
+  const Tab = createBottomTabNavigator();
+
+  return (
+    <Tab.Navigator
+      initialRouteName="Weather"
+      screenOptions={({route}) => ({
+        tabBarIcon: () => {
+          let iconName = 'ios-information-circle';
+          if (route.name === 'Weather') {
+            iconName = 'cloud';
+          } else if (route.name === 'Settings') {
+            iconName = 'gear';
+          }
+          return <Icon name={iconName} size={20} color="white" />;
```

Comment:
let's also use color from yours 'styles/themeColors' instead of 'white' (line 25, 26 as well)